app/Http/Controllers/api/CommentController.php

```diff
@@ -0,0 +1,137 @@
+<?php
+
+namespace App\Http\Controllers\api;
+
+use App\Models\Comment;
+use App\Models\User;
+use App\Models\Post;
+use App\Http\Controllers\Controller;
+use Illuminate\Http\Request;
+use App\Http\Requests\Api\CommentCreatRequests;
+use App\Http\Resources\Api\CommentResource;
+use App\Http\Requests\Api\CommentCreatRequest;
+
+
+class CommentController extends Controller
+{
+  public function store(CommentCreatRequests $request, $postId)
+  {
+    //return $postId;
+    //$path = $request->file('a')->store('public');
+    //  return $path;
+    // $user = $request->user();
+    // $post = $user->posts()->create([
+    //     'title' => $request->title,
+    //     'contente' => $request->contente,
+    //     'image' => $request->image,
+    //     'likes' => 0,
+    //     'isPublished' => 1,
+    // ]);
+
+    $Comment = Comment::create([
+
+      'user_id' => $request->user()->id,
+      'post_id' => $postId,
+      'comment' => $request->comment,
+    
+    ]);
```

Comment:
$comment = $post->comments()->create([
      'user_id' => $request->user()->id,
       'comment' => $request->comment,
])

Reply:
return  "message": "Call to a member function comments() on string",